src/GatewayService.php

```diff
@@ -291,11 +293,11 @@ function GenerateXsell(GatewayRequest $request, GatewayResponse $response)
 //
 //////////////////////////////////////////////////////////////////////
 //	
-  function BuildPaymentLink($request, $response): bool
+  function BuildPaymentLink($request, $response)
   {
     $request->Set("gatewayServlet", "/hostedpage/servlet/BuildPaymentLinkSubmit");
     $this->PerformTransaction($request, $response);
-    return ($response->Get(GatewayResponse::RESPONSE_CODE()) == GatewayCodes__RESPONSE_SUCCESS &&
+    return ($response->Get(GatewayResponse::RESPONSE_CODE()) == 0 &&
```

Comment:
@Nick-RG, please explicitly use `GatewayCodes::RESPONSE_SUCCESS`, instead of `0`.